controllers/aggregated_clusteroperator_controller.go

```diff
@@ -87,8 +86,8 @@ func (a *AggregatedClusterOperatorReconciler) Reconcile(ctx context.Context, req
 	// any failing status states, and update the aggregate CO resource
 	// to reflect those failing PO resources.
 	if statusErrorCheck := util.InspectPlatformOperators(poList); statusErrorCheck != nil {
-		coBuilder.WithAvailable(openshiftconfigv1.ConditionFalse, utilerror.NewAggregate(statusErrorCheck.FailingErrors).Error(), "POError")
-		return ctrl.Result{}, nil
+		coBuilder.WithAvailable(openshiftconfigv1.ConditionFalse, statusErrorCheck.Error(), "POError")
+		return ctrl.Result{}, statusErrorCheck
```

Comment:
Forgot to revert this change. I realized this was causing hotloop-esq behaviors, and we already established the requisite watches to get any new information.